instrumentation/sequel/lib/sequel/extensions/opentelemetry.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'sequel'
+
+# Sequel: The Database Toolkit for Ruby
+module Sequel
+  # The Sequel::Opentelemetry extension
+  #
+  # @example Simple usage
+  #
+  #     require 'sequel-opentelemetry'
+  #     DB.extension :opentelemetry
+  #
+  # Related module: Sequel::Opentelemetry
+  module Opentelemetry
+    def tracer
+      @tracer ||= ::OpenTelemetry.tracer_provider.tracer('Sequel', ::Sequel::VERSION)
+    end
+
+    def trace_execute(super_method, sequel_method, sql, options, &block)
+      response = nil
+      attributes = {
+        'component' => 'Sequel',
```

Comment:
Why did you call this `component`? Is this supposed to be used as a tag to find all spans related to this instrumentation?

Reply:
Yes. Is there a better semantic naming for that?

Reply:
I do not know off the top of my head. Would you mind reaching out to the folks in CNCF or opening an issue https://github.com/open-telemetry/semantic-conventions/issues? 

https://cloud-native.slack.com/archives/C01QZFGMLQ7
https://cloud-native.slack.com/archives/C041APFBYQP

Reply:
On second thought, i think that this should really already be covered by `InstrumentationScope`. 

That is a resource attribute attached to all spans and is currently named `OpenTelemetry::Instrumentation::Sequel`. 

Would that be sufficient?

Reply:
![image](https://github.com/open-telemetry/opentelemetry-ruby-contrib/assets/170848/67e3fed9-27b3-40d9-96ea-fa88e36530f9)
Yep, that's fine.